kubernetes-extension/src/main/ballerina/ballerinax/kubernetes/annotation.bal

```diff
@@ -93,6 +93,20 @@ public type ConfigMapKeyRef record {
     !...
 };
 
+@final public ImagePullPolicy IF_NOT_PRESENT = "IfNotPresent";
+@final public ImagePullPolicy ALWAYS = "Always";
+@final public ImagePullPolicy NEVER = "Never";
+
+# Image pull policy type field for kubernetes deployment and jobs.
+public type ImagePullPolicy "IfNotPresent"|"Always"|"Never";
+
+@final public RestartPolicy ON_FAILURE_RESTART = "OnFailure";
```

Comment:
We cannot use these variables as of yet as they are not values during compile time.  We can only have these now:

public type RestartPolicy "OnFailure"|"Always"|"Never";

We have to wait for compile time constants. Then we can define such values.

Reply:
Let's for now just define the type, so people can give the literal string value for it at least there in the annotation, and it will be properly documented on the supported values.